.storybook/stories/AlertBanner/AlertBanner.tsx

```diff
@@ -19,12 +21,34 @@ export const AlertBanner: FC<AlertBannerProps> = ({
   onClose,
 }) => {
   const styles = useStyles(variant);
+  let opacity = new Animated.Value(AlertBannerConsts.FROM);
+
+  const close = () => {
+    animation(
+      opacity,
+      getEasingFunc('ease'),
+      AlertBannerConsts.TO,
+      AlertBannerConsts.FROM,
+      AlertBannerConsts.CLOSE_DURATION,
+    ).start(onClose);
+  };
+
+  useEffect(() => {
+    animation(
+      opacity,
+      getEasingFunc('ease'),
+      AlertBannerConsts.FROM,
+      AlertBannerConsts.TO,
+      AlertBannerConsts.START_DURATION,
+    ).start(() => setTimeout(close, AlertBannerConsts.CLOSE_ALERT));
```

Comment:
Now this timeout isn't in `start` callback so it will start immediately to countdown not after the animation. Check it it's ok. Idk how to clean up this timeout in `start` callback. If you know how to solve it better, let me know.

Reply:
I used `userRef` to solvethis problem